MOD/Witcher/history/characters/155000_special.txt

```diff
@@ -63,21 +63,7 @@
 	1269.1.1={ 
 		employer=45001
 	}
-	1274.7.1 = {
-		effect = {
-			c_oxenfurt = {
-				holder_scope = {
-					ROOT = { 
-						imprison = PREV
-					}
-				}
-			}
-		}
-	}
-	1275.1.1 = {
-		death = yes
-		death_reason = death_dungeon
-	}
+	1274.6.1 = { death = yes }

```

Comment:
Ok that makes sense as "Left Síle de Tansarville to die." is the default preset in the "Witcher 3" (which can be considered "canon") according http://witcher.wikia.com/wiki/The_Witcher_3_decision_checklist